Formula/quilt.rb

```diff
@@ -31,6 +36,10 @@ def install
     if OS.mac?
       args << "--with-sed=#{HOMEBREW_PREFIX}/bin/gsed"
       args << "--with-stat=/usr/bin/stat" # on macOS, quilt expects BSD stat
+      if MacOS.version >= :ventura
+        args << "--with-diff=#{HOMEBREW_PREFIX}/bin/diff"
+        args << "--with-patch=#{HOMEBREW_PREFIX}/bin/patch"
```

Comment:
This doesn't work if not linked. `#{Formula["diffutils"].opt_bin}` would be a better location.

Reply:
Doesn't that go for `patch` and `gsed` too?

Reply:
Yes it does